addons/modules/config.cpp

```diff
@@ -4,6 +4,7 @@ class CfgPatches {
     class ADDON {
         name = COMPONENT_NAME;
         units[] = {
+            QGVAR(moduleAddFullArsenal),
```

Comment:
Maybe just `moduleAddArsenal` since there is only adding/removing full arsenals.

Reply:
But down the road, a new module could be added for a limited arsenal and then you'd have to change this again.

Reply:
I guess you want to make a distinction between what kind of Arsenal is placed for the players and what kind of Arsenal is used when Zeus is editing a loadout.

Reply:
Sry, my last comment did not fit here.
I would keep the name `moduleAddFullArsenal` based on @CreepPork argument.